src/main/resources/db/changelog/changes/v2.1/add_authority_indices.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+
+  <changeSet id="MODELINKS-141@@create-authority-heading_type-index" author="Mukhiddin_Yusupov">
```

Comment:
I suggest to create these indexes to only authority_archive table, as for now this API will be used to retrieve deleted authorities.